scripts/emerge/deprecated/create_file_ingest_json.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+import pandas as pd
+
+
+def create_file_ingest_json(input_file):
+    """Create newline delimited json file for TDR bulk file ingest API."""
+
+    ingest_files = pd.read_excel(input_file, sheet_name="Sheet1", index_col=None)
+
+    output_basename = input_file.split(".xlsx")[0]
+    output_json = open(f"{output_basename}_newline_delimited.json", "w")
+
+    for row in ingest_files.iterrows():
+        row[1].to_json(output_json)
```

Comment:
why are you grabbing the second item in `row` ?

Reply:
`iterrows()` returns an object (Series, I think) with the index of each row and the data in each row so the second item is the data for each row. Probably could just handle this without needing this but I stopped developing this script early on. I put it into a deprecated folder just to not have to delete it in case I wanted to reference it for code snippets.